pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer._test.go

```diff
@@ -0,0 +1,73 @@
+package jobruntestcaseanalyzer
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+)
+
+func TestGetJobs(t *testing.T) {
+
+	tests := map[string]struct {
+		allJobs          []jobrunaggregatorapi.JobRow
+		expectedJobNames sets.String
+		filters          string
+	}{
+		"test upgrade filter":   {expectedJobNames: sets.String{"name-without-it": sets.Empty{}, "name-with-something-else": sets.Empty{}}, filters: "upgrade"},
+		"test no filter":        {expectedJobNames: sets.String{"name-without-it": sets.Empty{}, "name-with-something-else": sets.Empty{}, "name-with-upgrade": sets.Empty{}}},
+		"test multiple filters": {expectedJobNames: sets.String{"name-without-it": sets.Empty{}}, filters: "upgrade,something"},
```

Comment:
Nit, but it's clearer to me and also IMHO generally a better test if the cases look more like the real world data.  Can we use real job names here?

Reply:
Good thought and I agree, thanks.